code/modules/mob/living/ventcrawling.dm

```diff
@@ -55,15 +55,9 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, typecacheof(list(
 			if(!client)
 				return
 
-			if(iscarbon(src) && ventcrawler < 2)//It must have atleast been 1 to get this far
-				var/failed = 0
+			if(iscarbon(src) && ventcrawler == VENTCRAWLER_NUDE)
 				var/list/items_list = get_equipped_items(include_pockets = TRUE)
-				if(items_list.len)
-					failed = 1
-				for(var/obj/item/I in held_items)
-					failed = 1
-					break
-				if(failed)
+				if(items_list.len || held_items.len)
```

Comment:
This is incorrect held items is always length of available hands.

Reply:
THEN WHY WAS THIS MERGED OH GOD

Reply:
this was already fixed upstream and i am no longer panicking